lib/omniauth/strategies/dfe_openid_connect.rb

```diff
@@ -0,0 +1,27 @@
+# This strategy ensures that the id_token_hint param is included in the post_logout_redirect_uri.
+# The node-oidc-provider library requires this param to be present in order for the redirect to work.
+# See: https://github.com/panva/node-oidc-provider/blob/main/lib/actions/end_session.js#L27
+# See: https://github.com/omniauth/omniauth_openid_connect/blob/master/lib/omniauth/strategies/openid_connect.rb#L423
```

Comment:
Nit: would it be worth replacing main/master with the SHAs of current HEAD in both cases, so the comment remains accurate as time goes on?